docs/features/reuse.md

```diff
@@ -0,0 +1,29 @@
+# Reusable Containers (Experimental)
+
+!!! warning 
+    Reusable containers is still an experimental feature and the behavior can change.
+    Those containers won't stop after all tests are finished.
+
+Reusable containers enables to keep containers running. In order
+to achieve it, manual initialization should be used by calling `start()`
+and should be enabled by environment. In order to reuse a container, the
+configuration *must not change*.
+
+!!! note
+    Reusable containers doesn't suit for CI usage and as an experimental feature
+    not all Testcontainers features are working such as cleanup, parallelization
```

Comment:
what's "parallelization" in this context?

Reply:
I wanted to reference to `ℹ️ Locking - two parallel tests with the same hash will reuse the same container` as mentioned in the [original PR](https://github.com/testcontainers/testcontainers-java/pull/1781) but I think I get it wrong and should be dropped.